package.json

```diff
@@ -7,7 +7,7 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test": "yarn test:unit && yarn test:e2e",
-    "lint": "vue-cli-service lint && prettier --write \"./*.{ts,js,json,md,vue}\" \"./src/**/*.{ts,js,json,md,vue}\" \"./__tests__/**/*.{ts,js,json,md,vue}\"",
+    "lint": "vue-cli-service lint --no-fix \"./*.{ts,js,json,md,vue}\" \"./src/**/*.{ts,js,json,md,vue}\" \"./__tests__/**/*.{ts,js,json,md,vue}\"",
```

Comment:
re-add `prettier` to the `lint` command as it's now unused. Also make sure eslint fixes the errors that it can